src/city/culture.c

```diff
@@ -191,6 +210,17 @@ void city_culture_calculate(void)
         }
     }
     if (num_houses) {
+        
+        city_data.culture.average_theater /= num_houses;
+        city_data.culture.average_amphitheater /= num_houses;
+        city_data.culture.average_colosseum /= num_houses;
+        city_data.culture.average_hippodrome /= num_houses;
+        city_data.culture.average_school /= num_houses;
+        city_data.culture.average_library /= num_houses;
+        city_data.culture.average_academy /= num_houses;
+        city_data.culture.average_hospital /= num_houses;
+        city_data.culture.average_religion_x10 = (city_data.culture.average_religion * 10)  / num_houses;
```

Comment:
Instead of creating this strange `average_religion_x10`, can't you calculate the average religion directly in `ratings.c` by using `city_data.culture.religion_coverage` (which is already normalized to a `0-100` format)?